src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,34 @@
+import * as moment from "moment";
+import * as React from "react";
+import * as css from "./Notification.scss";
+
+interface Props {
```

Comment:
ive been putting I in front of all interfaces, e.g. IProps, so lets stick with that